CHANGELOG.md

```diff
@@ -7,9 +7,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased](https://github.com/pusher/push-notifications-swift/compare/0.10.10...HEAD)
 
+### Added
+
+- Retry strategy with exponential backoff.
+
+### Removed
+
+- NetworkResponse.
+
+### Changed
+
+- Moved the DispatchQueue inside the class. Queue takes care of orchestrating the network requests so they are executed in sequential order.
```

Comment:
Same thing. If you want this information, you can just check the commit history. The CHANGELOG should be more about answering the question: "Should I upgrade?" or "What useful things have happened that might make me want to upgrade".

That said, we can still have a more generic "Refactored/Improved Network layer internals".

What do you think?